### DISCUSSION TITLE
Use pnpm

### DISCUSSION THREAD

**Discussion by k-taro56** [indigo-san/mita-memo#5] [Ideas]
Corepack 経由で pnpm を使う
Bun でもいいかもしれないが、Runtime から変わっちゃうので任せる

See also:
https://zenn.dev/monicle/articles/1c06f3f75b2cb1
👆 この記事のこれも See also:
https://zenn.dev/teppeis/articles/2021-05-corepack

**Comment by yuto-trd** [comment]
https://github.com/indigo-san/mita-memo/commit/939bce2ec15d8c8d7d1a98a0dbb852a44f2586a2

**Reply by k-taro56** [reply]
ええやん
